.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  printWidth: 100,
+  parser: 'flow',
+  tabWidth: 2,
+  singleQuote: true,
+  bracketSpacing: true,
+  arrowParens: 'always',
+  trailingComma: 'es5',
+  semi: true,
+};
```

Comment:
why did you move this file to `.js`?

Reply:
To be consistent with .eslint.js and we can add comments and stuff